torch/_refs/__init__.py

```diff
@@ -4201,8 +4201,10 @@ def tensor_split(
             )
             raise ValueError(msg)
         if indices_or_sections.dtype != torch.long:
-            msg = "tensor_split: if indices_or_sections is a tensor it must have long dtype, "
-            f" but received one with dtype {indices_or_sections.dtype}"
+            msg = (
+                "tensor_split: if indices_or_sections is a tensor it must have long dtype, "
```

Comment:
Technically, these are implicit concatenations, but yeah definitely cleaner this way.

Reply:
it's a subtle bug, there's no concatenation without `\` on the first line.